### ISSUE TITLE
Add an option for executing timeout

### ISSUE THREAD

**Issue by iapYang** [nimroddolev/chime_tts#49]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/nimroddolev/chime_tts/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

My chime_tts runs, and it generates the cache file, but it stops because of error

```shell
 Service call to chime_tts.say timed out after 15 seconds.
``` 

### Describe the solution you'd like

Because of the machine which I'm running homeassistant isn't powerful enough

I notice in the source code, the 15 second timeout is constant

```python
result = await asyncio.wait_for(task, 15)
``` 
maybe can provide a property `timeout`  in options to overwrite the default 15 seconds

### Describe alternatives you've considered

Not applicable

### Additional context

Not applicable

**Comment by nimroddolev** [comment]
Hi @iapYang, that you for your feedback. I have included in the latest version (v0.9.0) an option to adjust the timeout in the configuration dialog for Chime TTS:

![chime_tts_options copy](https://github.com/nimroddolev/chime_tts/assets/1849295/c08f59b7-e9d7-4f18-a0e4-9f49a30f792f)


You can view the configuration dialog by navigating to:
`Settings` --> `Devices & services` --> `Chime TTS` --> `CONFIGURE`

Please let me know if this addresses the issue 👍

**Comment by iapYang** [comment]
@nimroddolev thanks a lot, it works! I'm gonna close this issue.
